### PR TITLE
PLAT-146 added first test encrypt file and protect

### PR DESCRIPTION
**Pull request by benlstanley**
https://app.saucelabs.com/tests/e3ec8de5dcd84edb93d53018c3fbeb66#3 - Passed test in Sauce Labs

### CODE REVIEW

**Review by jrschumacher — COMMENTED** [review]
@benlstanley where are the step definitions?

**Comment by benlstanley** [comment]
@jrschumacher https://virtru.atlassian.net/wiki/spaces/QA/pages/156827675/Framework+v3+BDD+Step+Implementations
Currently step definitions are added manually here, but Ken is hoping to automate this in due time.

**Comment by jrschumacher** [comment]
Looks like it meets all the step definitions

**Review by jrschumacher — APPROVED** [review]

File: e2e/tests/features/sdk_website.feature [hunk 1]
```diff
@@ -15,16 +13,19 @@ Feature: SDK Demo Site
...
+  @smoke
+  Scenario Outline: Encrypt file in Protect and Track demo site
+    Given an attachment file: <filename>
     When I start to run testrail <case_id>
-    And qavirtru31@cmk-test-org.com logged in secure-reader page using a new browser
-    And the user drag and drop the file in secure-reader page
-    And the file automatically decrypt in secure-reader page
-    Then screen comparison shall have similarity(0.0-1.0) greater than 0.999
+    And I opened protect-and-track page using a new browser
+    And I drag and drop the file in protect-and-track page
```
Comment (jrschumacher) [inline e2e/tests/features/sdk_website.feature]: @benlstanley according to the site `the user drag and drop the file in (secure-reader, gmail, protect-and-track) page`

Comment (benlstanley) [reply e2e/tests/features/sdk_website.feature]: Ok good catch we will update the confluence page!

**Pull request merged**